de/api/configuration-ignore.md

```diff
@@ -3,6 +3,35 @@ title: "API: The ignore Property"
 description: Define the ignore files for your Nuxt.js application
 ---
 
+# .nuxtignore
+
+You can use `.nuxtignore` to let Nuxt.js building phase ignore `layout`, `page`, `store` and `middleware` files in your project’s root directory (`rootDir`). The `.nuxtignore` file is file wih same specification like `.gitignore` and `.eslintignore` in which each line is a glob pattern indicating which files should be ignores.
```

Comment:
```suggestion
You can use a `.nuxtignore` file to let Nuxt.js ignore `layout`, `page`, `store` and `middleware` files in your project’s root directory (`rootDir`) during the build phase.
The `.nuxtignore` file is subject to the same specification as `.gitignore` and `.eslintignore` files, in which each line is a glob pattern indicating which files should be ignored.
```

Reply:
Some info that'd be good to have in there:

* Is the `.nuxtignore` file only interpreted/found when it's in the project root directory (as .gitignore files can be nested in subfolders)
* Are there CodeSandbox examples?

Reply:
Yes. Only works for root dir.
Example needs to be added